includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```
@@ -529,6 +532,9 @@ public function javascript_params() {
 			$stripe_params['excludedPaymentMethodTypes']   = $this->get_excluded_payment_method_types();
 		}
 
+		// Adaptive Pricing feature flag and setting.
+		$stripe_params['isAdaptivePricingSupported'] = WC_Stripe_Feature_Flags::is_checkout_sessions_available() && 'yes' === $this->get_option( 'adaptive_pricing', 'no' );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Feature_Flags' in method 'javascript_params'. (undefined)

(StaticAccess)